fishtank/src/cluster.ts

```diff
@@ -96,53 +97,25 @@ export class Cluster {
     const node = new Node(this, options.name)
     const containerName = naming.containerName(this, options.name)
 
-    const runOptions: RunOptions = {
+    const runOptions = {
+      args: ['start', ...(options.extraArgs ?? [])],
       name: containerName,
       networks: [naming.networkName(this)],
       hostname: options.name,
       labels: { [CLUSTER_LABEL]: this.name, ...options.extraLabels },
+      volumes: new Map<string, string>(),
     }
 
-    const args: string[] = ['start', ...(options.extraArgs ?? [])]
-
-    const dest = node.dataDir
-    if (options.config || options.networkDefinition) {
-      await promises.mkdir(dest, {
-        recursive: true,
-      })
-    }
+    await promises.mkdir(node.dataDir, { recursive: true })
+    runOptions.volumes.set(node.dataDir, CONTAINER_DATADIR)
```

Comment:
I see in your test you have added also `tmpdir(), 'fishtank',` to the runOptions that are called with. Is that added elsewhere not in this code change?